CreeDictionary/CreeDictionary/relabelling.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import annotations
+
+import csv
+from enum import IntEnum
+from typing import Iterable, Optional, TextIO, Tuple, TypeVar, Union
+
+from utils import shared_res_dir
+from utils.types import FSTTag, Label
+
+CRK_ALTERNATE_LABELS_FILE = shared_res_dir / "crk.altlabel.tsv"
+
+Default = TypeVar("Default")
+
+
+class _LabelFriendliness(IntEnum):
+    """
+    Weird enum that I'm not sure should have ever existed.
+
+    Values are assigned according to their corresponding column in crk.altlabel.tsv
+    """
+
+    LINGUISTIC_SHORT = 1
+    LINGUISTIC_LONG = 2
+    ENGLISH = 3
+    NEHIYAWEWIN = 4
+    EMOJI = 5
+
+
+class Relabelling:
+    """
+    Given an FST tag, provides access to the relabellings, as written by the linguists
+    (mostly Antti).
+
+    Access as the following:
+
+        .linguistic_short[tag]  or .linguistic_short.get(tag, default)
+        .linguistic_long[tag]   or .linguistic_long.get(tag, default)
+        .english[tag]           or .english.get(tag, default)
+        .cree[tag]              or .cree.get(tag, default)
+        .emoji[tag]             or .emoji.get(tag, default)
+    """
+
+    # This data structure is kind of a weird, but I'm keeping it for now.
+    _DataStructure = dict[Tuple[FSTTag, ...], dict[_LabelFriendliness, Optional[Label]]]
```

Comment:
**MUST** use `Tuple` from typings in this context due to this issue: https://github.com/python/mypy/issues/9522

Using `tuple[FSTTag, ...]` yields an `unexpected ...` error from mypy.